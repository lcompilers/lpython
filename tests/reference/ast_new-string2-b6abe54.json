{
    "basename": "ast_new-string2-b6abe54",
    "cmd": "lpython --show-ast --indent --new-parser --no-color {infile} -o {outfile}",
    "infile": "tests/parser/string2.py",
    "infile_hash": "58397718a6e0dc5cc453081e82ad1b2ca378639dfb516c3f28d34323",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_new-string2-b6abe54.stdout",
    "stdout_hash": "52d1edc0e332527fc6e077ed895f9d01a6eb79b3fb43405319864a1c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}