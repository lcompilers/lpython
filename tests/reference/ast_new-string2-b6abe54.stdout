(Module
    [(Expr
        (ConstantStr
            "\"\\"
            ()
        )
    )
    (Expr
        (ConstantStr
            "\n\\"
            ()
        )
    )
    (Expr
        (ConstantStr
            "\\"
            ()
        )
    )
    (Expr
        (ConstantStr
            "   \\"
            ()
        )
    )
    (Expr
        (ConstantStr
            "\\ "
            ()
        )
    )
    (Expr
        (ConstantStr
            ""
            ()
        )
    )
    (Expr
        (ConstantStr
            "\\|"
            ()
        )
    )
    (Expr
        (ConstantStr
            "I said 'Wow' "
            ()
        )
    )
    (Expr
        (ConstantStr
            "I said \"hi\" "
            ()
        )
    )
    (Expr
        (ConstantStr
            "I said \\\"hi\" "
            ()
        )
    )
    (Expr
        (ConstantStr
            "I said '\\'hi\\hey' "
            ()
        )
    )
    (Expr
        (ConstantStr
            "ab"
            ()
        )
    )
    (Expr
        (ConstantStr
            "ab"
            ()
        )
    )
    (Expr
        (ConstantStr
            "Text\\\\"
            ()
        )
    )
    (Expr
        (ConstantStr
            ""
            ()
        )
    )
    (Expr
        (Call
            (Name
                x
                Load
            )
            [(ConstantStr
                "class a:\n    def test():\n        return r\"\\LaTeX\"\n"
                ()
            )]
            []
        )
    )
    (Expr
        (ConstantStr
            "\n    \"\\\\\"\n"
            ()
        )
    )
    (Expr
        (ConstantStr
            "\n    '\\'\n"
            ()
        )
    )]
    []
)
